cms/models.py

```diff
@@ -72,6 +74,27 @@ def save(self, *args, **kwargs):
         super(LearningOutcomesPage, self).save(*args, **kwargs)
 
 
+class LearningTechniquesPage(CourseProgramChildPage):
+    """
+    Teaching techniques page for learning.
+    """
+
+    subpage_types = []
+    technique_items = StreamField(
+        [("techniques", LearningTechniqueBlock())],
+        blank=False,
+        help_text="Enter detail about how you'll learn.",
+    )
+
+    content_panels = [StreamFieldPanel("technique_items")]
+
+    def save(self, *args, **kwargs):
+        # auto generate a unique slug so we don't hit a ValidationError
+        self.title = "Learning Techniques"
+        self.slug = slugify("{}-{}".format(self.get_parent().id, self.title))
+        super(LearningTechniquesPage, self).save(*args, **kwargs)
```

Comment:
The pattern in the rest of the codebase outside of the CMS code that you guys have worked on is to use the python 3 style `super`: `super().save(*args, **kwargs)`. Unless that doesn't work for some reason, please change that here

Reply:
Strange, It is working fine now after remove `page-container` class. I was seeing a bar earlier that's why I added. Ok I am going to remove it.